src/scss/pages/carousel.scss

```diff
@@ -234,16 +235,31 @@
   }
 }
 @media (max-width: 768px) {
+  .carousel-container {
+    height: 300px;
+    .section__title{
+    font-size: 3.0em;
+    padding-bottom: 5%;
+  }
+}
+
   .alice-carousel__stage-item img {
    position: relative;
    right: -63px;
-  }
+    }
```

Comment:
the wrong indent again... If you decided to skip my comment about code quality controller for styles, you should to create a task in trello for this to do this later and to notify a reviewer about your decision, but not ignore any comments, please...